connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,588 @@
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNameApiKeyAuth"
+      }
+    },
+    {
+      "label": "Multiple login",
+      "group": "authentication",
+      "description": "Allow or disallow multiple login. For more information on multi-login, see <a href=\"https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/\" target=\"_blank\">documentation</a>",
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        {
+          "name": "TRUE",
+          "value": "true"
+        },
+        {
+          "name": "FALSE",
+          "value": "false"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.multipleLogin"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNamePasswordAuth"
+      }
+    },
+    {
+      "label": "Token",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "token"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "tokenAuth"
+      }
+    },
+    {
+      "description": "Token",
+      "type": "Hidden",
+      "value": "={\"X-Authorization\":token}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "headers"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "tokenAuth"
+      }
+    },
+    {
+      "description": "Output headers",
+      "type": "Hidden",
+      "value": "={\"X-Authorization\":\"asObject:token.asString\"}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.outputHeaders"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Authentication URL",
+      "description": "Full authentication url. For more information, see <a href=\"https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/\" target=\"_blank\">documentation</a>",
+      "type": "String",
+      "group": "configuration",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.url"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^(=|http://|https://|secrets).*(authentication)$",
```

Comment:
as mentioned, here it would be great to reuse the  control room @igpetrov . If the bug blocks us here, please create a follow up ticket @Oleksiivanov